lib/elixir/lib/string.ex

```diff
@@ -201,6 +201,9 @@ defmodule String do
       iex> eacute
       233
 
+  See the documentation for [`<<>>`](`<<>>/1`) or the Patterns and Guards
+  guide for more information on binary pattern matching
```

Comment:
I *think* you can link to the guide as well:

```suggestion
  See the documentation for [`<<>>`](`<<>>/1`) or the [*Patterns and Guards*
  guide](patterns-and-guards.html) for more information on binary pattern matching.
```

Can you try out `make docs_elixir` locally with this and see if the link is to the right page?

Reply:
`.md` perhaps?
Another instance from `Kernel` doc:

```
[Patterns and guards](patterns-and-guards.md)
```

Reply:
Ah sure yep

Reply:
done. built the docs and it links to the correct pages